etc/helm/pachyderm/values.yaml

```diff
@@ -713,6 +716,9 @@ oidc:
   mockIDP: true
   #TODO scopes:
 
+testConnection:
+  image: alpine
```

Comment:
can we make this `image: {repository: ..., tag: ...}` to be consistent with the other image specifications?

Reply:
yup